src/templates/status.html

```diff
@@ -16,6 +16,9 @@
                         if (result.indexOf(key) != -1) {
                             $("#status-" + runId).html("<i class=\"fas fa-sync-alt\"></i> " + result)
```

Comment:
Let's move this line in the else branch of the if statement underneath. I noticed that the button text is updated twice when I tested it. First time it's updated by this line, the second time updated by `getQueuePosition` function. This gives the illusion that the status is _continuously_ being updated. We should make sure this button's content only changed once so that students know they still need to click on it multiple times.